AzureBot/Dialogs/ActionDialog.cs

```diff
@@ -35,18 +35,29 @@ public class ActionDialog : LuisDialog<string>
 
             context.Wait(MessageReceived);
         }
-
+        [LuisIntent("Hello")]

```

Comment:
Can't have a new intent here.  Just need to map "hello" utterance to the Help intent.